tasks/main.yml

```diff
@@ -10,3 +10,5 @@
 
 - include: inputs.yml
   when: not graylog_skip_inputs_installation
+
+- include: plugins.yml
```

Comment:
graylog_skip_plugins_installation for example, should be added in defaults/main.yml